src/app/components/chart/areaStackedTimeSeries.chart.directive.js

```diff
@@ -29,11 +29,18 @@
             scope.legend = true;
 
             var chart;
+            
+            $rootScope.$on("angular-resizable.resizeEnd", function (event, args) {
+                var element = document.getElementById(args.id);
+                var child = element.childNodes[3].childNodes[0].childNodes[0].childNodes[3];
```

Comment:
This is prone to error. If anything changes in the widget template, this will break. Also, it's not exactly flexible. Vector can have multiple widget templates and this will error if the template is different than the default.